oc_ldap_user_locker/locker.py

```diff
@@ -142,7 +143,7 @@ def _compare_attribute(self, values, match_conf):
         logging.debug("Finall check: returning '%s'" % str(_result))
         return _result
 
-    def _check_user_conf(self, user_rec, conf):
+    def _check_user_conf(self, user_rec, conf, ldap_c):
```

Comment:
Who is `ldap_c`? Please add it to docstring method specification below (at least).
But we are moving it from method-to-method, and that is enough to make it class member: `self.ldap_c`